datasette/app.py

```diff
@@ -440,6 +478,10 @@ def metadata(self, key=None, database=None, table=None, fallback=True):
                 m.update(item)
             return m
 
+    @property
+    def _metadata(self):
+        return self.metadata()
+
```

Comment:
If this is for backwards compatibility I don't think it's necessary - `_metadata` has never been part of Datasette's public documented API.